homeassistant/components/modbus/base_platform.py

```diff
@@ -86,13 +88,25 @@ def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
     async def async_update(self, now=None):
         """Virtual function to be overwritten."""
 
-    async def async_base_added_to_hass(self):
-        """Handle entity which will be added."""
-        if self._scan_interval > 0:
-            cancel_func = async_track_time_interval(
+    @callback
+    async def async_remote_start_stop(self, force_stop):
```

Comment:
This doesn't need to be a coroutine function since we don't await inside.

Reply:
Done.